transport_nantes/press/templates/press/press_create.html

```diff
@@ -3,6 +3,10 @@
 {% block content %}
 <div class="container-fluid pl-5 py-3">
     <div class="col-md-8 col-sm-12 mx-auto">
+      <a href="{% url 'press:list_items' %}" class="btn navigation-button mb-2">
+        <i class="fa-solid fa-arrow-left"></i>
```

Comment:
We probably shouldn't use `fa-arrow-left` when we mean to cancel.  It's really for going back.  I'll make an issue.